python/foglamp/services/core/api/filters.py

```diff
@@ -74,8 +76,16 @@
         if category_info is not None:
             raise ValueError("This '{}' filter already exists".format(filter_name))
 
-        # Load C filter plugin info
-        loaded_plugin_info = apiutils.get_plugin_info(plugin_name, dir='filter')
+        # Load C/Python filter plugin info
+        #loaded_plugin_info = apiutils.get_plugin_info(plugin_name, dir='filter')
+        try:
+            # Try fetching Python filter
+            plugin_module_path = "{}/python/foglamp/plugins/filter/{}".format(_FOGLAMP_ROOT, plugin_name)
```

Comment:
@pintomax  https://github.com/foglamp/FogLAMP/commit/46a78659c7dbc91089bfc4dde236085d62bdcc32

Otherwise it will result in errors!